Actions/DetermineArtifactUrl/action.yaml

```diff
@@ -16,32 +16,20 @@ inputs:
   settingsJson:
     description: Settings from repository in compressed Json format
     required: false
-    default: '{"appBuild":0,"appRevision":0}'
   secretsJson:
     description: Secrets from repository in compressed Json format
     required: false
     default: '{"insiderSasToken":""}'
-outputs:
-  ArtifactUrl:
-    description: The ArtifactUrl to use for building this project
-    value: ${{ steps.determineArtifactUrl.outputs.ArtifactUrl }}
-  ArtifactCacheKey:
-    description: The Artifact Cache Key to use for building this project (if using CompilerFolder)
-    value: ${{ steps.determineArtifactUrl.outputs.ArtifactCacheKey }}
 runs:
   using: composite
   steps:
-    - name: Determine ArtifactUrl
+    - name: run
       shell: ${{ inputs.shell }}
-      id: determineArtifactUrl
       env:
         _parentTelemetryScopeJson: ${{ inputs.parentTelemetryScopeJson }}
         _project: ${{ inputs.project }}
-        _settingsJson: ${{ inputs.settingsJson }}
         _secretsJson: ${{ inputs.secretsJson }}
-      run: try { 
-          ${{ github.action_path }}/DetermineArtifactUrl.Action.ps1 -parentTelemetryScopeJson $env:_parentTelemetryScopeJson  -project $ENV:_project -settingsJson $ENV:_settingsJson -secretsJson $ENV:_secretsJson
-        } catch { Write-Host "::Error::Unexpected error when running action ($($_.Exception.Message.Replace("`r",'').Replace("`n",' ')))"; exit 1 }
+      run: try { ${{ github.action_path }}/DetermineArtifactUrl.ps1 -parentTelemetryScopeJson $env:_parentTelemetryScopeJson -project $ENV:_project -settingsJson $ENV:Settings -secretsJson $ENV:_secretsJson } catch { Write-Host "::Error::Unexpected error when running action ($($_.Exception.Message.Replace("`r",'').Replace("`n",' ')))"; exit 1 }
```

Comment:
Why do we need to pass the _settingsJson_ parameter when the script can directly read the settings from `$env:Settings`?

For instance, in Actions/DetermineArtifactUrl/DetermineArtifactUrl.ps1,
the line
```
$projectSettings = $settingsJson | ConvertFrom-Json | ConvertTo-HashTable
```
(`$settingsJson` is the parameter here) may simply become
```
$projectSettings = $env:Settings | ConvertFrom-Json | ConvertTo-HashTable
```

Reply:
done